Books/client/src/features/misc/routes/Landing.tsx

```diff
@@ -0,0 +1,80 @@
+import digitalMinimalism from '../../../assets/images/digital_minimalism.webp';
+import readingNook from '../../../assets/images/readingNook.svg';
+import './Landing.scss';
+
+export const Landing = () => {
+  return (
+    <>
+      <nav className="flex justify-end">
+        <button className="secondary-btn">Sign In</button>
+        <button className="primary-btn">Sign Up</button>
+      </nav>
+      <section className="flex justify-between section-container">
+        <div className="hero__text">
+          <h1 className="h1 heading">Find your next best book and read it all day.</h1>
```

Comment:
I think that h1 should be font-weight 400, to look like on design. Doesn't really matter actually, just was comparing to figma design.

Reply:
Just fixed it.